common/headers/caller_info.go

```diff
@@ -33,6 +34,10 @@ var (
 		CallerName: CallerNameSystem,
 		CallerType: CallerTypePreemptable,
 	}
+	SystemOptionalCallerInfo = CallerInfo{
+		CallerName: CallerNameSystem,
+		CallerType: CallerTypeOptional,
+	}
```

Comment:
I think you want this new priority to be higher priority than CallerTypePreemptable because things like retention timer, delete visibility task are also using CallerTypePreemptable.

Reply:
I see. My plan was to keep the current priority. Do you think the mentioned operation can be lower priority than live replication?

Reply:
add replication type